File: 프로그래머스/0/181926. 수 조작하기 1/수 조작하기 1.java
class Solution {
    public int solution(int n, String control) {
        
        for(Character c : control.toCharArray()){
            if(c=='w'){
                n++;
            }else if(c=='s'){
                n--;
            }else if(c=='d'){
                n += 10;
            }else if(c=='a'){
                n -= 10;
            }else {
                System.out.println("조건에 없는 문자"); 
            }
        }
        
        return n;
    }
}

File: 프로그래머스/0/181926. 수 조작하기 1/수 조작하기 1.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int n, string control) {
    int answer = 0;
    for(char c : control) {
        if(c == 'w'){
            n++;
        } else if (c=='s') {
            n--;
        } else if (c=='d') {
            n+=10;
        } else if (c=='a') {
            n-=10;
        }
    }
    answer = n;
    return answer;
}